posthog/clickhouse/log_entries.py

```diff
@@ -0,0 +1,80 @@
+from posthog.clickhouse.kafka_engine import KAFKA_COLUMNS, kafka_engine, ttl_period
+from posthog.clickhouse.table_engines import ReplacingMergeTree
+from posthog.kafka_client.topics import KAFKA_LOG_ENTRIES
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+LOG_ENTRIES_TABLE = "log_entries"
+LOG_ENTRIES_TTL_DAYS = 90
+
+LOG_ENTRIES_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    team_id UInt64,
+    -- The name of the service or product that generated the logs.
+    -- Examples: batch_exports
+    log_source LowCardinality(String),
+    -- An id for the log source.
+    -- Set log_source to avoid collision with ids from other log sources if the id generation is not safe.
+    -- Examples: A batch export id, a cronjob id, a plugin id.
+    log_source_id String,
+    -- An id for the instance of log_source that generated this log.
+    -- This may be ommitted if log_source is a singleton.
+    -- Examples: A batch export run id, a plugin_config id, a thread id, a process id, a machine id.
+    instance_id String,
```

Comment:
I find these names a bit confusing ... I think the key here is that if you only use one it should be clear which one to use, so from that perspective it could be `primary_id`/`main_id` & `secondary_id`/`optional_id` for example.

Reply:
I agree with the spirit: I saw `log_source_id` as a `primary_id` and `instance_id` as a `secondary_id. However naming is hard, so I would just ask for a vote

Reply:
I don't really care about these names besides the minor inconvenience of having to update my PR. Since my table has become *our* table, if the people commenting here feel the name should be something else I'll make it something else.

Reply:
I think once we have several sources of logs the naming here will be clearer and we can migrate with a rename if needed

You _could_ change the comment to `-- A secondary id e.g. for the instance of log_source that generated this log.`

As a compromise position here... but I'm excited to start building on top of this